src/spans/awsSpan.test.js

```diff
@@ -113,6 +113,12 @@ describe('awsSpan', () => {
     expect(awsSpan.getBasicSpan()).toEqual(expectedBasicSpan);
   });
 
+  test('getBasicSpan turn is warm', () => {
+    expect(utils.isWarm()).toEqual(false);
+    awsSpan.getBasicSpan();
```

Comment:
Why `getBasicSpan()` should change it to be `'warm'`?

Reply:
for the next time, you run the tracer, it was the behavior that Sagi wants, he just had a bug